src/cplus_plugin/gui/constant_raster_manager_dialog.py

```diff
@@ -543,7 +578,7 @@ def _ensure_checked_components_in_collection(self):
 
         # Get the current value from the widget to use for ALL checked items
         current_value = None
-        if hasattr(current_config_widget, 'sb_experience'):
+        if hasattr(current_config_widget, "sb_experience"):
```

Comment:
@osundwajeff this seems to be tightly coupled to the years of experience in the raster dialog, which should be generic without metadata-specific implementations at this level.